.github/workflows/build-for-release.yml

```diff
@@ -9,37 +9,50 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        build_name:
+          [linux-x64-musl, linux-arm64-musl, macos-arm64, windows-x64-msvc]
         include:
-          - os: ubuntu-latest
+          - build_name: linux-x64-musl
+            os: ubuntu-latest
             exe_suffix: ""
             name_suffix: ""
             target: x86_64-unknown-linux-musl
-          - os: macos-latest
+          - build_name: linux-arm64-musl
+            os: ubuntu-latest
+            exe_suffix: ""
+            name_suffix: ""
```

Comment:
`name_suffix` in all builds can be removed, it was only used for the asset name

Reply:
Are you sure? I think it is used here:

```
asset_name: dotter-${{ matrix.build_name}}${{ matrix.name_suffix }}
```

Reply:
i missed it 🙃